src/main/kotlin/racing/View.kt

```diff
@@ -0,0 +1,26 @@
+package racing
+
+object InputView {
+    fun readCarCount(): Int {
+        println("자동차 대수는 몇 대인가요?")
+        return readlnOrNull()?.toIntOrNull() ?: 0
```

Comment:
Kotlin의 [Safe calls](https://kotlinlang.org/docs/null-safety.html#safe-calls)를 잘 활용하셨네요! 👍👍 

앞에서 이미 Safe Call을 하고있기 때문에 chaning 함수는 그냥 `toInt()`를 사용하셔도 무방할 것 같습니다! (아래도 동일해요!)
```suggestion
        return readlnOrNull()?.toInt() ?: 0
```

Reply:
toIntOrNull() 은 숫자가 아니면 null 을 반환하는 반면, toInt() 는 숫자가 아니면 NumberFormatException 예외를 던지네요.
이 부분은 밑에 코멘트 주신 것 (문자 검증) 까지 합해서 같이 수정해볼게요 🙂

Reply:
아앗... 그렇군요. 그 부분은 생각을 못했어요. 😅 
당연히 숫자값만 올 것이라고 생각하면서 코멘트를 남겨놨었는데, 아예 숫자가 아닌 값이들어왔을 때, 발생하는 예외는 생각을 못했네요. 🤔